config/ember-try.js

```diff
@@ -98,6 +98,18 @@ module.exports = {
       resolutions: {
         "ember": "canary"
       }
+    },
+    {
+      name: 'ember-data-2.3',
+      dependencies: {
+        "ember": "components/ember#release",
+      },
+      devDependencies: {
+        "ember-data": "~2.3"
+      },
+      resolutions: {
+        "ember": "release"
+      }
```

Comment:
Hmm, I just realized that ember-try only works with bower dependencies for now. So I am not sure this config is really helpful to test the code path below :pensive: